src/appleseed/renderer/modeling/camera/camera.cpp

```diff
@@ -80,13 +80,16 @@ bool Camera::on_render_begin(
     IAbortSwitch*           abort_switch)
 {
     m_shutter_open_time = m_params.get_optional<float>("shutter_open_time", 0.0f);
-    m_shutter_open_end_time = m_params.get_optional<float>("shutter_open_end_time", 0.2f);
-    m_shutter_close_start_time = m_params.get_optional<float>("shutter_close_start_time", 0.8f);
+    m_shutter_open_end_time = m_params.get_optional<float>("shutter_open_end_time", 0.0f);
```

Comment:
Do you confirm that instantaneous shutter opening and closing are properly handled by `map_to_shutter_curve()` (ideally without much overhead)?

Reply:
Yes. When shutter open instantly, sample doesn't map to 1st case of `map_to_shutter_curve()`, and it is similar with 2nd case and closing instantly case. I could check it works well.